src/pages/library/datasetBuilder/Breadcrumbs.ts

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash/fp';
+import { Fragment } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { Link } from 'src/components/common';
+import * as Utils from 'src/libs/utils';
+
+// This is a new pattern that design wants us to introduce instead of the current breadcrumb pattern used in the headers
+interface Breadcrumb {
+  title: string;
+  link: string;
+}
+
+export const DatasetBuilderBreadcrumbs = ({ breadcrumbs }: { breadcrumbs: Breadcrumb[] }) =>
```

Comment:
I've been told to always use a Props type and use destructuring for component methods
```suggestion
type DatasetBuilderBreadcrumbsProps = { breadcrumbs: Breadcrumb[]; };

export const DatasetBuilderBreadcrumbs: React.FC<DatasetBuilderBreadcrumbsProps> = (props) => {
  const { breadcrumbs } = props;
```